### PR TITLE
Profile and abstract

### PR DESCRIPTION
**Pull request by dijital20**
* Added the talk abstract.
* Updated the profile for Josh Schneider

### CODE REVIEW

File: docs/past_meetups/posts/2023-09-11-josh.md
Comment (MasonEgger) [inline docs/past_meetups/posts/2023-09-11-josh.md]: Can we add a little more here? Like a one sentence topic sentence on what you're going to cover? Don't worry too much about below the fold, we'll update that after the presentation.

Comment (dijital20) [reply docs/past_meetups/posts/2023-09-11-josh.md]: I merged the teaser sentence with the content below. Let me know what you think of that, or if you are looking for even more detail. @bradsbrown , let me know what you think too.

Comment (bradsbrown) [reply docs/past_meetups/posts/2023-09-11-josh.md]: I think that with it all pulled together it looks solid to me. @MasonEgger's the one with write access though! 😂

Comment (dijital20) [reply docs/past_meetups/posts/2023-09-11-josh.md]: @bradsbrown Thanks man! I know @MasonEgger is out on vacation, so I wanted a second opinion in case I needed to rework the content before he gets back.

Thanks for reviewing!

**Review by MasonEgger — APPROVED** [review]
LGTM

**Pull request merged**
